MapboxVisionAR/src/main/java/com/mapbox/vision/ar/view/gl/GlRender.kt

```diff
@@ -18,8 +16,8 @@ class GlRender(
     private val height: Int
 ) : GLSurfaceView.Renderer {
 
-    interface OnSurfaceChangedListener {
-        fun onSurfaceChanged()
+    interface Renderer {
```

Comment:
'GLSurfaceView.Renderer' already exist and it spells really similar. May be rename 'Renderer' to 'RenderListener' or 'RenderEventListener'?

Reply:
Well I did it on purpose, since the interface actually works with the exact same purpose as GLSurfaceView.Renderer. I plan to add `draw()` call there, too.

Reply:
`draw` requires different data in different cases. So I don't think it's a good idea to attach `draw` to interface.

Reply:
Draw just calls the actual redraw of textures/triangles etc, data can be (guess even should) set separately. There is no need to count and pass `camera` and `lane` params every time frame is drawn, if it has not changed.

Reply:
get it